presentation/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,9 @@
     <uses-permission android:name="android.permission.INTERNET"/>
     <uses-permission android:name="android.permission.READ_MEDIA_IMAGES"/>
     <uses-permission android:name="android.permission.RECEIVE_BOOT_COMPLETED"/>
-
+    <uses-permission android:name="android.permission.SCHEDULE_EXACT_ALARM"/>
+    <uses-permission android:name="android.permission.USE_EXACT_ALARM"/>
```

Comment:
I was getting a cash so I had to add these permissions. Is this correct?
``` Caller me.androidbox.presentation needs to hold android.permission.SCHEDULE_EXACT_ALARM or android.permission.USE_EXACT_ALARM to set exact alarms```

Reply:
Yes, `USE_EXACT_ALARM` is needed here

Reply:
Yes. Also im pretty sure that having UseExactAlarm, you need to ask for that permission at runtime also

For emulators and so on, its not needed
But necessary if you plan to upload to playstore